src/vs/workbench/test/browser/workbenchTestServices.ts

```diff
@@ -545,6 +545,7 @@ export class TestMenuService implements IMenuService {
 export class TestHistoryService implements IHistoryService {
 
 	declare readonly _serviceBrand: undefined;
+	declare shouldIgnoreActiveEditorChange: boolean;
```

Comment:
I think this is not very safe, this would not actually put the property on the class as far as I know but just declare it as type to be there.